dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -347,6 +354,17 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
         return m_useButtonStyle ? L" useButtonStyle='true'" : L"";
     }
 
+    std::wstring AppNotificationBuilder::GetProgressBars()
+    {
+        std::wstring result{};
+        for (auto progressBar : m_progressBarList)
+        {
+            result.append(progressBar.as<winrt::Windows::Foundation::IStringable>().ToString());
```

Comment:
Can you use sprintf? string.append is expensive

Reply:
We agreed to leave as is for now and address later if possible.